spec/support/requirements_helper.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+%i[form summary].each do |style|
+  RSpec::Matchers.define :"have_#{style}_item" do |field, key, context = {}|
```

Comment:
I found the naming for these matcher somewhat confusing. Maybe something more explicit like `have_requirements_form_text`? As we seem to only be checking the translation and 'form/summary' is distinguished at an "item" level.

Reply:
I agree 'text' would better reflect what we're actually testing. However, I can see it being confusing when written in full e.g. `have_requirements_form_text(:access_limit, :user_not_in_orgs)`, as I would expect the argument to be a simple string.

We could directly test there is an issue by checking the 'field' and 'key' attributes, which reflects the args we're passing in the matcher. This would end up being a generic 'have_issue' matcher, but we could still superficially cover translations existing:

```
# how does this look?
expect(issues).to have_issue(:alt_text, :blank, styles: %i[form summary])
```

If we still want to validate the text, which has some benefit if it contains interpolated args, then it would help to understand better what it is that's confusing for you. Can you explain what it is about the current name that's confusing?

Reply:
Ah yup - I like your suggestion! I think it would also allow us to add future checks for an issue.

I found the original name `have_#{style}_item` confusing because the reference `<style>_item` doesn't map to any concept in the codebase.

Reply:
@kevindew how does `have_issue` sound to you?

Reply:
Sounds good 👍